src/components/ListItem.css

```diff
@@ -12,3 +13,23 @@
 .ListItem-label {
 	margin-left: 0.2em;
 }
+
+.indicator-soon {
+	color: orange;
+}
+
+.indicator-kind-of-soon {
+	color: yellow;
```

Comment:
I think that the yellow may not have enough color contrast from an accessibility standpoint.

Here is a screenshot when analyzing with Wave 

![screenshot](https://cdn.zappy.app/443fad8376654f97ae5850c2c2e260bd.png)

Reply:
We decided to choose our colors based on how the site looks with a dark background, which seemed to be the default based on the current CSS. We can add an alternative to the colors for light backgrounds with a media query.